src/components/Accordion/Stories/Accordion.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import Accordion from '../Accordion'
+import AccordionItem from '../AccordionItem';
+
+export default {
+    title: "Components/Accordion",
+    component: Accordion,
+    subcomponents : {AccordionItem},
+    argTypes: {
+        children: {
+            control: "none",
+            description: "`<AccordionItem />` elements."
+        }
+    }
+
+} as ComponentMeta<typeof Accordion>;
+
+const TemplateIcons: ComponentStory<typeof Accordion> = (args) => (
+    <Accordion {...args} />
+);
+export const AccordionElement = TemplateIcons.bind({});
+AccordionElement.args = {
+    children : [
+        <AccordionItem label="Lorem ipsum label">
```

Comment:
Add story for `AccordionItem` and use it as children content in story for the `Accordion` element. You can have a look in the `Card` story for an example how to do it.

Reply:
@haschek  Thanks for the information I will do it in the same way